example.js

```diff
@@ -1,30 +1,21 @@
-const { collector } = require("./build");
-const { join } = require("path");
+const { collector } = require('./build');
+const { join } = require('path');
 
 (async () => {
-  const EMULATE_DEVICE = "iPhone 13 Mini";
+    const URL = 'amazon.com';
```

Comment:
What's the thought behind this change? I'm guessing you wanted to change it to something that actually returned some results?

Reply:
Can we change this back to example.com for now? I get wanting to change it to something with results but I want to think a bit more about if we want to do that, and if so what it should be. One of the problems with making it amazon.com is some people expect to see a lot more tracking and think something is broken when there's not. See https://themarkup.org/the-breakdown/2020/10/01/i-scanned-my-favorite-social-media-site-on-blacklight-and-it-came-up-pretty-clean-whats-going-on.

Reply:
Amazon actually has a ton of tracking, but I've changed it back. It's not super important.